deployment/seed-data.sh

```diff
@@ -2,10 +2,28 @@
 
 set -euxo pipefail
 
-newman run \
-  --folder "Publish Master Data" \
-  --env-var data_management_url="http://$EDC_HOST:9191/api/v1/data" \
-  --env-var storage_account="$ASSETS_STORAGE_ACCOUNT" \
-  --env-var participant_id="$PARTICIPANT_ID" \
-  --env-var api_key="$API_KEY" \
-  deployment/data/MVD.postman_collection.json
\ No newline at end of file
+function split_by_comma() {
```

Comment:
`entrypoint.sh` uses a different method to split, would be good to align to make it easier to maintain for non-bash experts

Reply:
How about similar to `entrypoint.sh` using a single input string `company1:company1assets,company2:company1assets`... (two of the inputs having redundant values `company1`, 2, 3)

The bash code here is pretty complex

Reply:
@algattik using a single input like `company1:company1assets,company2:company1assets...` will cause the issue for cloud deployment as the same shell script is used by cloud deployment data seeding as well and their EDC_HOST and PARTICIPANT_ID are different.

Reply:
Updated script to use similar input and delimiter as `entrypoint.sh`.